component/Post/index.js

```diff
@@ -10,7 +10,7 @@ import { Header } from "component/Post/Header";
  */
 export default function PostComponent(props) {
     const { post } = props;
-    return <div className='flex flex-col items-center h-screen bg-white md:w-1/2 lg:w-1/3 xl:1/4 overflow-hidden'>
+    return <div className='flex flex-col items-center h-dvh bg-white md:w-1/2 lg:w-1/3 xl:1/4 overflow-hidden'>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from 'h-screen' to 'h-dvh' might cause issues if the class 'h-dvh' is not defined in your Tailwind CSS configuration. Please ensure that this class is properly defined and behaves as expected.
```diff
- return <div className='flex flex-col items-center h-screen bg-white md:w-1/2 lg:w-1/3 xl:1/4 overflow-hidden'>
+ return <div className='flex flex-col items-center h-dvh bg-white md:w-1/2 lg:w-1/3 xl:1/4 overflow-hidden'>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->